tasks/collect-marketplacesvm-secret/collect-marketplacesvm-secret.yaml

```
@@ -42,3 +45,5 @@ spec:
         fi
 
         jq -j '.mapping.cloudMarketplacesSecret' "$DATA_FILE" | tee "$(results.cloudMarketplacesSecret.path)"
+        jq -jr '.mapping | if has("cloudMarketplacesPrePush") then .cloudMarketplacesPrePush else "false" end' \
```

Can we not do this? Seems simpler. Also, `-j` and `-r` don't make sense together - `-j` is enough.
```suggestion
        jq -j '.mapping.cloudMarketplacesPrePush // false' \
```

---

I've tried that but it seems it fails when the key is absent...

---

Why would it fail? We use that in many places. For example:
```
$ jq -j '.mapping.cloudMarketplacesPrePush // false' <<< '{}'
false
```

---

Nevermind.. It's working now, I'm patching it

---

Fixed! It's working 😄